src/ResourceManager/Search/Commands.Management.Search/ChangeLog.md

```diff
@@ -0,0 +1,22 @@
+<!--
```

Comment:
This is comments for the above psd1 file:
- Version should be 0.1.0
- When you create the format.ps1xml please put the link in FormatsToProcess
- All modules coming into our repo start off as preview modules.  This allows you to make breaking changes based on customer feedback before becoming a stable module that must follow our 6 month breaking change policy.  To indicate that this is a preview module, please add the preview tag: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Aks/Commands.Aks/AzureRM.Aks.psd1#L117